src/newrelic/app.go

```diff
@@ -294,7 +295,7 @@ func combineEventConfig (ehc collector.EventHarvestConfig, sehc collector.SpanEv
 }
 
 func parseConnectReply(rawConnectReply []byte) (*ConnectReply, error) {
-	var c ConnectReply
+	c := ConnectReply{MaxPayloadSizeInBytes: limits.DefaultMaxPayloadSizeInBytes}
 
```

Comment:
Do we need to worry about if the connect reply had a value > 1000*1000 or does the spec say to accept a larger value if returned?